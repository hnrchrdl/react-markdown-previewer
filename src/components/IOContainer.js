import React from 'react';
import styled from 'styled-components';

const Container = styled.div`
  width: 50%;
`;

export default props => <Container>{props.children}</Container>;
