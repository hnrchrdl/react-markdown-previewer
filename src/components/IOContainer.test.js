import React from 'react';
import IOContainer from './IOContainer';
import renderer from 'react-test-renderer';

test('App renders and matches snapshot', () => {
  const component = renderer.create(<IOContainer />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
