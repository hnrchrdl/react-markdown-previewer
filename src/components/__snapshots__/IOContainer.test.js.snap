// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders and matches snapshot 1`] = `
<div
  className="sc-bdVaJa fgQBgM"
/>
`;
